src/main/java/org/example/algorithms/Recursion/CalcPower.java

```diff
@@ -0,0 +1,19 @@
+public class CalcPower {
```

Comment:
Package info at the top of each class is missing.

```java 
package org.example.algorithms.recursion;

```

See other classes for references.

- Use lowercase for package names. Thanks.